docs/_docs/reference/experimental/into.md

```diff
@@ -0,0 +1,285 @@
+```
+However, the next section shows that unwrapping with `.underlying` is not needed for parameters, which is the most common use case. So explicit unwrapping should be quite rare.
+
+
+
+### Dropping `into` for Parameters in Method Bodies
+
+The typical use cases for `into` wrappers are for parameters. Here, they specify that the
+corresponding arguments can be converted to the formal parameter types. On the other hand, inside a method, a parameter type can be assumed to be of the underlying type since the conversion already took place when the enclosing method was called. This is reflected in the type system which erases `into` wrappers in the local types of parameters
+as they are seen in a method body. Here is an example:
+```scala
+  def ++ (elems: into[IterableOnce[A]]): List[A] =
+    val buf = ListBuffer[A]()
+    for elem <- elems.iterator do // no `.underlying` needed here
+      buf += elems
+    buf.toList
+```
+Inside the `++` method, the `elems` parameter is of type `IterableOnce[A]`, not `into[IterableOne[A]]`. Hence, we can simply write `elems.iterator` to get at the `iterator` method of the `IterableOnce` class.
+
+Specifically, we erase all `into` wrappers in the local types of parameter types that appear in covariant or invariant position. Contravariant `into` wrappers are kept since these typically are on the parameters of function arguments.
+
+### Into Constructors in Type Aliases
+
+Since `into` is a regular type constructor, it can be used anywhere, including in type aliases and type parameters. For instance, in the Scala standard library we could define
+```scala
+type ToIterator[T] = into[IterableOnce[T]]
+```
+and then `++`, `flatMap` and other functions could use this alias in their parameter types. The effect would be the same as when `into` is written out explicitly.
+
+## Second Scheme: `into` as a Modifier
+
+The `into` scheme discussed so far strikes a nice balance between explicitness and convenience. But migrating to it from Scala 2 implicits does require major changes since possibly a large number of function signatures has to be changed to allow conversions on the arguments. This might ultimately hold back migration to Scala 3 implicits.
+
+To facilitate migration, we also introduce an alternative way to specify target types of implicit conversions. We allow `into` as a soft modifier on
+classes, traits, and opaque type aliases. If a type definition is declared with `into`, then implicit conversions into that type don't need a language import.
+
+For instance, the Laminar framework
+defines a trait `Modifier` that is commonly used as a parameter type of user-defined methods and that should support implicit conversions into it.
+`Modifier` is commonly used as a parameter type in both Laminar framework functions and in application-level functions that use Laminar.
+
+We can support implicit conversions to `Modifier`s simply by making `Modifier` an `into` trait:
+```scala
+into trait Modifier ...
```

Comment:
how would Laminar maintainers preview integrating the feature without having to make typical users set experimental flag? I guess create a new branch with the mod and release under special version

Reply:
That would be a way to do it. I agree our experimental scheme does not lend itself well to multi-group experiments. Of course Laminar could turn the flag on for themselves and use it in their code base and tests. That would already tell them something.